include/megaapi.h

```diff
@@ -18285,6 +18318,25 @@ class MegaApi
         */
         void sendBackupHeartbeat(MegaHandle backupId, int status, int progress, int ups, int downs, long long ts, MegaHandle lastNode);
 
+        /**
+         * @brief Allow fetch google ads
+         *
+         * @param adFlags A bitmap flag used to communicate with the API
+         * @param adUnits A list of the adslot ids to fetch
+         * @param publicHandle Provide the public handle that the user is visiting (optionally)
+         * @param listener MegaRequestListener to track this request
+         */
+        void fetchGoogleAds(int adFlags, MegaStringList *adUnits, MegaHandle publicHandle, MegaRequestListener *listener = nullptr);
+
+        /**
+         * @brief Allows you to simply query if we should show an ad or not
```

Comment:
```suggestion
         * @brief Check if Google ads should show or not
```
Also, missing doc about the request. And we probably should include some context and possible values for `adFlags`.